contracts/token/ERC721/IERC721.sol

```diff
@@ -6,48 +6,109 @@ import "../../introspection/IERC165.sol";
  * @dev Required interface of an ERC721 compliant contract.
  */
 interface IERC721 is IERC165 {
+    /**
+     * @dev Emitted when `tokenId` token is transfered from `from` to `to`.
+     */
     event Transfer(address indexed from, address indexed to, uint256 indexed tokenId);
+
+    /**
+     * @dev Emitted when `owner` enables `approved` to manage the `tokenId` token.
+     */
     event Approval(address indexed owner, address indexed approved, uint256 indexed tokenId);
+
+    /**
+     * @dev Emitted when `owner` enables or disables (`approved`) `operator` to manage all of its assets.
+     */
     event ApprovalForAll(address indexed owner, address indexed operator, bool approved);
 
     /**
-     * @dev Returns the number of NFTs in ``owner``'s account.
+     * @dev Returns the number of tokens in `owner`'s account.
      */
     function balanceOf(address owner) external view returns (uint256 balance);
 
     /**
-     * @dev Returns the owner of the NFT specified by `tokenId`.
+     * @dev Returns the owner of the `tokenId` token.
      */
     function ownerOf(uint256 tokenId) external view returns (address owner);
 
     /**
-     * @dev Transfers a specific NFT (`tokenId`) from one account (`from`) to
-     * another (`to`).
-     *
-     *
+     * @dev Safely transfers `tokenId` token from `from` to `to`.
+     * 'Safely' means that, if `to` refers to a smart contract, the token won't be forever locked in it.
+     * It ensures that the receiving contract knows how to properly handle ERC721 tokens by checking if it returns the correct function selector.
```

Comment:
```suggestion
     * @dev Transfers `tokenId` token from `from` to `to`, checking first that contract recipients
     * are aware of the ERC721 protocol to prevent tokens from being forever locked.
```